taichi/analysis/clone.cpp

```diff
@@ -12,7 +12,7 @@ TLANG_NAMESPACE_BEGIN
 class IRCloner : public IRVisitor {
  private:
   IRNode *other_node;
```

Comment:
Lovely, wonder what `clang-tidy` criteria is for picking which set of private members to fix...

Reply:
Huh, both False negatives and false positives seem to be inevitable for static analysis tools.

Reply:
quite interesting that the one in taichi/analysis/same_statements.cpp has been picked up :))

Reply:
It could also be that sources outside of the src tree referenced this and since tidy only changes the specified source tree, it disabled this. Only a theory, not sure whether it's the actual case here.